### ISSUE TITLE
Does it save bandwidth while using this app???

### ISSUE THREAD

**Issue by Halamadrid777** [Ashish-Bansal/audio-only-youtube#73]

**Comment by theRealBitcoinClub** [comment]
yes

**Comment by colin-guyon** [comment]
Checking the dev tools network tab, the requests still download webm content. Shouldn't it be another type of content? Anyway I don't feel the size of these packets is lighter than when playing video in 144p. Do I make a mistake, or did you noticed that yourself?
Thanks :)

**Comment by MulverineX** [comment]
YouTube's high-quality audio format is Opus, which is usually contained in either `.ogg` or `.webm`. `googlevideo`, youtube's content CDN, distributes Opus audio standalone in a webm container, with the video in a separate stream/file. tldr; webm can contain just audio, youtube's does, this extension does save bandwidth/performance (no video encoding).

**Comment by RyszardGmail** [comment]
Yes, but You will trip with issues on videos which originally don't have video track in it - music with static image.
